emscripten.py

```diff
@@ -2301,7 +2301,7 @@ def debug_copy(src, dst):
   # tell binaryen to look at the features section, and if there isn't one, to use MVP
   # (which matches what llvm+lld has given us)
   cmd += ['--detect-features']
-  if shared.Settings.DEBUG_LEVEL >= 2 or shared.Settings.PROFILING_FUNCS:
```

Comment:
Should we find a way to make options available outside of emcc.py to avoid the need to all these extra settings in Settings.js?

Reply:
Perhaps. But a lot of the time we do need to use the new option inside the JS compiler too, and the settings.js mechanism is our current way to synchronize that world with python.